playbooks/roles/configure-ucsm/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+remove: false
```

Comment:
can we make this configure_ucsm_remove: false or remove_service_profile:false, "remove" by itself seems like something we could very quickly have a conflict with.

Reply:
done, we'll also need to make this change in the static-nexus-programmer role